viewer/public/index.html

```diff
@@ -52,7 +52,7 @@
     <!--VZV_HEADER_SNIPPETS-->
     <!-- DO NOT ALTER "ABOVE" THIS LINE FOR AUTOMATED ENVIRONMENT SNIPPETS -->
   </head>
-  <body style="background: #fff;">
+  <body style="background: #fff; overflow-x: hidden">
```

Comment:
The summary measures are overflowing their container and causing a horizontal scrollbar on certain display sizes. You can replicate this by removing this style property and setting your display size to 1366 x 1024. This is the rotated **iPad Pro** size on your chrome dev tools.

i spent quite a bit of time trying to fix the source of overflow in the DOM, but i ultimately gave up and decided to brute force it like this.

if we are uncomfortable apply this change globally, i initially used the `postMessage` API so that this change would only take place when the VZE loads the /measures view in an iFrame. Here is that code:
https://github.com/cityofaustin/vision-zero/pull/1663/commits/c9b8b2759456ea7fcbb15a8ca50d7f48f705f173